contracts/payloads/IGP24/PayloadIGP24.sol

```diff
@@ -0,0 +1,461 @@
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Approve fUSDC and fUSDT protocols to spend the reserves tokens
+    function action1() internal {
+        address[] memory protocols = new address[](2);
+        address[] memory tokens = new address[](2);
+        uint256[] memory amounts = new uint256[](2);
+
+        // fUSDC
+        protocols[0] = F_USDC;
+        protocols[0] = USDC_ADDRESS;
+        amounts[0] = 165_000 * 1e6; // 165k USDC
+
+        // fUSDT
+        protocols[1] = F_USDT;
+        protocols[1] = USDT_ADDRESS;
+        amounts[1] = 165_000 * 1e6; // 165k USDT
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+    }
+
+    /// @notice Action 2: closure of old vaults
+    function action2() internal {
+        address VAULT_ETH_USDC = 0x5eA9A2B42Bc9aC8CAC76E19F0Fcd5C1b06950807;
+        address VAULT_ETH_USDT = 0xE53794f2ed0839F24170079A9F3c5368147F6c81;
+        address VAULT_WSTETH_ETH = 0x28680f14C4Bb86B71119BC6e90E4e6D87E6D1f51;
+        address VAULT_WSTETH_USDC = 0x460143a489729a3cA32DeA82fa48ea61175accbc;
+        address VAULT_WSTETH_USDT = 0x2B251211f5Ff0A753A8d5B9411d736875174f375;
+
+        closeVault(VAULT_ETH_USDC, ETH_ADDRESS, USDC_ADDRESS);
+        closeVault(VAULT_ETH_USDT, ETH_ADDRESS, USDT_ADDRESS);
+        closeVault(VAULT_WSTETH_ETH, wstETH_ADDRESS, ETH_ADDRESS);
+        closeVault(VAULT_WSTETH_USDC, wstETH_ADDRESS, USDC_ADDRESS);
+        closeVault(VAULT_WSTETH_USDT, wstETH_ADDRESS, USDT_ADDRESS);
+    }
+
+    /***********************************|
+    |          Vault Helper             |
+    |__________________________________*/
+
+    function closeVault(address vault, address supplyToken, address borrowToken) internal {
+        // Set user supply config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserSupplyConfig[]
+                memory configs_ = new AdminModuleStructs.UserSupplyConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserSupplyConfig({
+                user: address(vault),
+                token: supplyToken,
+                mode: 1,
+                expandPercent: 0,
+                expandDuration: 1,
+                baseWithdrawalLimit: supplyToken == wstETH_ADDRESS ? 2 * 1e18 : 10 * 1e18 // 2 wstETH or 10 ETH
+            });
+
+            LIQUIDITY.updateUserSupplyConfigs(configs_);
+        }
+
+        // Set user borrow config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserBorrowConfig[]
+                memory configs_ = new AdminModuleStructs.UserBorrowConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserBorrowConfig({
+                user: address(supplyToken),
+                token: borrowToken,
+                mode: 1,
+                expandPercent: 0,
+                expandDuration: 1,
+                baseDebtCeiling: 10 * 1e6,
```

Comment:
can remove the 1e6 here even for both baseDebtCeiling and maxDebtCeiling. 

I just picked 10 and 100 back then to have some distance between the two values and they can not be 0. Theoretically we could also set 1 and 2 as the absolute lowest possible values, but I thought 10 and 100 is safer. Goal is just to have them very very low close to 0. 10 and 100 USD would also be low enough but just wanted to mention

Reply:
You mean 10 and 100 in wei?

Reply:
yes

Reply:
Done